Actions/DownloadProjectDependencies/README.md

```diff
@@ -6,7 +6,9 @@ The action constructs arrays of paths to .app files, that are dependencies of th
 ## INPUT
 
 ### ENV variables
-none
+| Name | Description |
+| :-- | :-- |
+| Settings | env.Settings must be set by a prior call to the ReadSettings Action |
```

Comment:
Maybe...

```suggestion
| Settings | env.Settings must be set by a prior call to the DownloadProjectDependencies Action |
```

Reply:
Nope - the only action setting env.Settings is ReadSettings.